src/components/movies/MovieCard.js

```diff
@@ -356,18 +356,18 @@ function MovieCard({
   const anchorRef = React.useRef(null);
   const [selectedIndex, setSelectedIndex] = React.useState(1);
 
-  const handleClickServiceProvider = () => {
-    console.info(`You clicked ${serviceProvider[selectedIndex]}`);
-  };
+  // const handleClickServiceProvider = () => {
+  //   console.info(`You clicked ${serviceProvider[selectedIndex]}`);
+  // };
 
```

Comment:
remove?

Reply:
will be used for service provider at a later date